trl/trainer/online_dpo_config.py

```
@@ -32,7 +32,9 @@ class OnlineDPOConfig(TrainingArguments):
             Initial learning rate for [`AdamW`] optimizer. The default value replaces that of
             [`~transformers.TrainingArguments`].
         reward_model_path (`Optional[str]`, *optional*, defaults to `None`):
-            Path to the reward model.
+            Path to the reward model. Either `judge` or `reward_model_path` must be set, but not both.
+        judge (`Optional[str]`, *optional*, defaults to `None`):
```

Should we raise an error in the post init of these configs if both are set?